app/src/main/java/src/ShapeL.java

```diff
@@ -1,9 +1,26 @@
 package src;
 
 public class ShapeL extends Shape {
-    @Override
-    public void draw() {
+    private static final int ROWS = 2;
+    private static final int COLUMNS = 3;
 
+    /**
+     * Draws the shape L into an array.
+     * @return An array of int with of the form of figure.
+     */
+    @Override
+    public int[][]  draw() {
+        int[][] mat = new int[ROWS][COLUMNS];
```

Comment:
This variable mat is not very descriptive. Maybe we can improve it.

Reply:
Thanks, Joel, I'm going to change it now

Reply:
It is already changing, I changed the name of the mat variable to matrix